requirements.txt

```diff
@@ -13,7 +13,7 @@ pylibemu==0.5.8
 pymongo==3.7.1
 python-magic==0.4.15
 rarfile==3.0
-requests==2.19.1
+requests==2.20.0
```

Comment:
This change is no longer required because the requirement was already updated in v0.9.37